lib/rpmsg/rpmsg_virtio.c

```diff
@@ -855,6 +867,10 @@ int rpmsg_init_vdev_with_config(struct rpmsg_virtio_device *rvdev,
 #endif /*!VIRTIO_DEVICE_ONLY*/
 
 	return status;
+
+err:
```

Comment:
to put under `#ifndef VIRTIO_DEVICE_ONLY` to fix build warning

Reply:
Thanks, done